README.md

```diff
@@ -126,6 +126,8 @@ lock-free tree on a lock-free pagecache on a lock-free log. the pagecache scatte
 partial page fragments across the log, rather than rewriting entire pages at a time
 as B+ trees for spinning disks historically have. on page reads, we concurrently
 scatter-gather reads across the log to materialize the page from its fragments.
+check out the [architectural outlook](https://github.com/spacejam/sled/wiki/sled-architectural-outlook)
```

Comment:
Maybe replace this with an inline link to make it more readable

Reply:
`wiki/...` doesn't work because it's a link to the wiki, not anything in this repo